hack/e2e/setup.sh

```
@@ -547,6 +561,49 @@ EOF
     _wait_rollout obs-mcp deployment/obs-mcp 3m
 }
 
+OPENSHIFT_MCP_SERVER_IMAGE_NAME="${OPENSHIFT_MCP_SERVER_IMAGE_NAME:-ghcr.io/rhobs/openshift-mcp-server}"
+OPENSHIFT_MCP_SERVER_IMAGE_REF="${OPENSHIFT_MCP_SERVER_IMAGE_REF:-${OPENSHIFT_MCP_SERVER_IMAGE_NAME}:${TAG:-latest}}"
+
+phase_deploy_oms() {
+    phase "deploy-oms"
+
+    step "Building openshift-mcp-server container image"
+    _run ${CONTAINER_CLI} build --load \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)